samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.multilingual.translation.model;
+
+/**
+ * Translation result from Translator API v3.
+ */
+public class TranslationResult {
+    private String text;
+
+    private String to;
+
+    /**
+     * Gets the translation result text.
+     * @return Translation result.
+     */
+    public String getText() {
+        return this.text;
+    }
+
+    /**
+     * Sets the translation result text.
+     * @param resultText Translation result.
+     */
+    public void setText(String resultText) {
+        this.text = resultText;
+    }
+
+    /**
+     * Gets the target language locale.
+     * @return Locale.
```

Comment:
why `locale`? How do you know if it's locale?

Reply:
[NodeJS Sample 17](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/javascript_nodejs/17.multilingual-bot/translation/microsoftTranslator.js#L21-L23) uses the `to` param and passes the locale, so it's the target locale